packages/contracts-bedrock/scripts/upgrades/holocene/.env.example

```diff
@@ -5,7 +5,7 @@
 # NOTE: The deploy config must be provided as a first argument to `just run`!
 
 # The network to deploy the contracts to.
-# Must be one of 'mainnet', 'sepolia'
+# Must be one of mainnet, sepolia
 NETWORK=
```

Comment:
When I used `NETWORK='sepolia'` this caused problems where the common.sh function which loads from the registry did not parse the network properly. I had to use `NETWORK=sepolia`.

Reply:
yes, the `.env` is parsed verbatim. I found out the hard way that `A="foo"` includes the `"`s in the variable.